From 4ee3aad605f16ba836a11e6807dd2ef3693faf6d Mon Sep 17 00:00:00 2001
From: Daniel P. Berrange <berrange@redhat.com>
Date: Thu, 15 Jun 2017 18:26:08 +0100
Subject: [PATCH] Use ./ when loading Virt.pm since "." is no longer in @INC

Latest Perl reports:

do "lib/Sys/Virt.pm" failed, '.' is no longer in @INC; did you mean do "./lib/Sys/Virt.pm"? at perl-Sys-Virt.spec.PL line 9.
lib/Sys/Virt.pm: No such file or directory at perl-Sys-Virt.spec.PL line 12.

Signed-off-by: Daniel P. Berrange <berrange@redhat.com>
---
 perl-Sys-Virt.spec.PL |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/perl-Sys-Virt.spec.PL b/perl-Sys-Virt.spec.PL
index aa3f6fc..dbb749d 100644
--- a/perl-Sys-Virt.spec.PL
+++ b/perl-Sys-Virt.spec.PL
@@ -6,7 +6,7 @@ use strict;
 
 die unless (scalar @ARGV == 1);
 
-unless (do 'lib/Sys/Virt.pm')
+unless (do './lib/Sys/Virt.pm')
   {
     if ($@) { die $@ };
     die "lib/Sys/Virt.pm: $!";
-- 
1.7.1

